QUICK_REFERENCE.md

```
@@ -0,0 +1,179 @@
+## 🔑 Credentials
+
+- **Host:** 45.159.160.5
+- **User:** esplzswx
+- **Pass:** q0Ju50iFb+m^6k]$
+- **Path:** /home/esplzswx/kanaanerpgaza-develop
+
+---
+
+## 🌐 Access Application
+
+- **URL:** http://45.159.160.5
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)